examples/Aggregator/Client/Client.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
does it make sense for examples to be switched over to net7.0 which is only in preview currently?
Wouldn't it make more sense to keep examples at the latest stable (net6.0) and only update our libraries to net7?

Reply:
It will be RC1 very soon. Changing examples back to net6.0 if someone wants is very easy.